### DISCUSSION TITLE
Would you mint telling me where the scrollbar colour is located?

### DISCUSSION THREAD

**Discussion by linusrg1** [rafaelmardojai/firefox-gnome-theme#427] [Q&A]
I am trying to modify this theme to be red.

**Comment by rafaelmardojai** [comment]
Actually, we don't theme the scrollbar.

**Comment by linusrg1** [comment]
> Actually, we don't theme the scrollbar.

Ah alright that must've been a firefox thing then. What about the little blue flash that animates across the tab once the page is loaded in?

**Comment by rafaelmardojai** [comment]
> Ah alright that must've been a firefox thing then. What about the little blue flash that animates across the tab once the page is loaded in?

That's also already present in Firefox.
